crates/gen/src/pwstr.rs

```diff
@@ -46,5 +46,19 @@ pub fn gen_pwstr() -> TokenStream {
                 ::windows::Param::Boxed(PWSTR(::std::boxed::Box::<[u16]>::into_raw(self.encode_utf16().chain(::std::iter::once(0)).collect::<std::vec::Vec<u16>>().into_boxed_slice()) as _))
             }
         }
+        #[cfg(windows)]
+        impl ::windows::IntoParam<'static, PWSTR> for &::std::ffi::OsStr {
```

Comment:
If `static` is "more correct" we should at least be consistent within the `pwstr.rs` source for all impls. Can you check that they have all been updated correctly to use the appropriate lifetime? I'm still learning about lifetimes...